code/datums/diseases/advance/symptoms/asthmosthia.dm

```diff
@@ -0,0 +1,40 @@
+/*
+//////////////////////////////////////
+asthmosthia
+
+	Very very Noticable.
+	Decreases resistance.
+	Decreases stage speed.
+	Reduced transmittability.
+BONUS
+	Spawns gases
+//////////////////////////////////////
+*/
+
+/datum/symptom/asthmothia
+
+	name = "Asthmothia"
+	stealth = -5
+	resistance = -5
```

Comment:
gunhog this shit SPITS PLASMA.

I'll see what I can do. I need to balance this.